Classes/Issues/IssuesViewController.swift

```diff
@@ -238,6 +241,9 @@ final class IssuesViewController:
                 y: messageView.frame.minY - manageButtonSize.height - Styles.Sizes.gutter),
             size: manageButtonSize
         )
+
+        // Recalculate the bottom content inset since the manageButton‘s visibility may have changed.
+        feed.collectionView.contentInset.bottom = threadInset.bottom
```

Comment:
I've been a little burned by mucking w/ insets when layout happens. Also the inset method requires that `isHidden` be set _before_ creating a new inset, which is done in `IssueManagingContextController`, so now we've created a hidden dependency. Really easy to break!

Maybe it should just be dependent on the `manageButton`'s height instead?

Reply:
Thanks for the feedback, @rnystrom. I agree with you about the hidden dependency and would like to find a way of avoiding it. 🙂

I'm not sure I understand your suggestion of making this depend on the button's height though. Right now, the height doesn't change when the button is hidden so relying on that to calculate the content inset wouldn't work (see line above where the size is set manually).

Reply:
Err ya you're right. Ignore me! Ha.

Reply:
How about putting a transparent view with the appropriate height at the bottom of the scroll view?

Reply:
@j-f1 and have it's height be determined by if the button is displayed?

Reply:
Yep!